lib/edge_commander/monitors/portable.ex

```diff
@@ -26,36 +28,52 @@ defmodule EdgeCommander.Portable do
     case get_current_status(nvr.nvr_id) do
       nil ->
         changeset_to_db(nvr, true)
-        update_nvr_status(nvr, true)
+        update_nvr_status(nvr, nil, true)
       false ->
         changeset_to_db(nvr, true)
-        update_nvr_status(nvr, true)
+        update_nvr_status(nvr, nil, true)
       true ->
         Logger.info "Status already true."
     end
   end
-  def port_status_to_db({:error, _}, nvr) do
+  def port_status_to_db({:error, reason}, nvr) do
     case get_current_status(nvr.nvr_id) do
       nil ->
         changeset_to_db(nvr, false)
-        update_nvr_status(nvr, false)
+        update_nvr_status(nvr, reason, false)
       true ->
         changeset_to_db(nvr, false)
-        update_nvr_status(nvr, false)
+        update_nvr_status(nvr, reason, false)
       false ->
         Logger.info "Status already false."
     end
   end
   # {:error, :nxdomain}
   # {:error, :timeout}
 
-  def update_nvr_status(nvr, status) do
-    IO.inspect status
+  def update_nvr_status(nvr, nil, status) do
     get_nvr!(nvr.nvr_id)
     |> Nvr.changeset(%{nvr_status: status})
     |> Repo.update!()
   end
 
```

Comment:
Remove this line.

Generally, when there are 2 methods with the same name just signatures change, they should be placed together.